src/main/java/juja/sqlcmd/Application.java

```diff
@@ -1,6 +1,153 @@
 package juja.sqlcmd;
 
+import java.sql.*;
+
 public class Application {
-    public static void main(String[] args) {
+
+    private static final String POSTGRESQL_DRIVER = "org.postgresql.Driver";
+    private static final String DB_CONNECTION_PATH = "jdbc:postgresql://127.0.0.1:5432/sqlcmd";
+    private static final String DB_USER = "sqlcmd";
+    private static final String DB_PASSWORD = "sqlcmd";
+    private static final String TABLE_NAME = "user";
+    private static final String LINE_SEPARATOR = System.lineSeparator();
+    private Connection connection;
+
+
+    public static void main(String[] arg) throws SQLException {
+        new Application().simpleSQL();
+    }
+
+    public void simpleSQL() throws SQLException {
+        checkJDBCDriver();
+        createDbConnection();
+        dropTableWithName(TABLE_NAME);
+        showExistingTablesName();
+        createTable(TABLE_NAME);
+        for (int i = 1; i < 4; i++) {
+            addUser("user" + i, "password" + i);
+        }
+        showExistingTablesName();
+        showExistingRecordsOfTable(TABLE_NAME);
+        updateDataByTitle("user2", "userchange1", TABLE_NAME, "name");
+        showExistingRecordsOfTable(TABLE_NAME);
+        deleteCortegeByTitle(TABLE_NAME, "name", "user3");
+        showExistingRecordsOfTable(TABLE_NAME);
+        connection.close();
+    }
+
+    private void deleteCortegeByTitle(String tableName, String columnName, String title) throws SQLException {
+        String sqlQuery = String.format("DELETE FROM \"%1$s\" WHERE %2$s = ? ", tableName, columnName);
+        try (PreparedStatement preparedStatement = connection.prepareStatement(sqlQuery)) {
+            preparedStatement.setString(1, title);
+            preparedStatement.executeUpdate();
+        }
+    }
+
+    private void updateDataByTitle(String oldData, String newData, String tableName, String columnName) throws SQLException {
+        String sqlQuery = String.format("UPDATE \"%3$s\" SET %4$s = \'%2$s\' WHERE %4$s = \'%1s\';", oldData, newData, tableName, columnName);
+        try (Statement statement = connection.createStatement()) {
+            statement.executeUpdate(sqlQuery);
+        }
+    }
+
+    private void showExistingRecordsOfTable(String tableName) throws SQLException {
+        try (Statement statement = connection.createStatement()) {
+            if (tableName != null) {
+                String sqlQuery = String.format("SELECT * FROM \"%s\"", tableName);
+                ResultSet resultSet = statement.executeQuery(sqlQuery);
+                int columnCount = resultSet.getMetaData().getColumnCount();
+
+                StringBuilder tableData = new StringBuilder();
+                while (resultSet.next()) {
+                    for (int i = 1; i <= columnCount; i++) {
+                        tableData.append(resultSet.getString(i)).append(" | ");
+                    }
+                    tableData.replace(tableData.length() - 3, tableData.length(), LINE_SEPARATOR);
```

Comment:
For easier code-reading it's better to extract tableData.length() to separate variable